kiliautoml/models/_ultralytics_object_detection.py

```diff
@@ -122,7 +129,12 @@ def train(
         data_path = os.path.join(model_repository_dir, "data")
         config_data_path = os.path.join(yolov5_path, "data", "kili.yaml")
 
-        if clear_dataset_cache and os.path.exists(data_path) and os.path.isdir(data_path):
+        if (
+            clear_dataset_cache
+            and os.path.exists(data_path)
+            and os.path.isdir(data_path)
+            and (not label_error_n_folds or (label_error_n_folds and label_error_cv_fold == 0))
```

Comment:
`not label_error_n_folds or (label_error_n_folds and label_error_cv_fold == 0)`  should be put into a variable with an explicit name.